### DISCUSSION TITLE
E1 Support?

### DISCUSSION THREAD

**Discussion by blumarten** [fwestenberg/reolink_dev#553] [Q&A]
I know the notes say that E1 and battery cameras aren't supported, (bummer, as these are the two types i have), but I see there are some discussions where people have some models of E1 working fine.

However whenever I try and enter the cam details in the integration set up the form just resets to blank and doesn't go any further.

Is it still the case E1s aren't supported at all?

**Comment by martinberteig** [comment]
This isn't a fault of the integration but from Reolink.
The only E1 models that support RTSP streaming are the E1 Pro and the E1 Zoom (i have two E1 Zoom cameras and they work perfectly)

Reolink support mentions RTSP support for these cameras:
Applies to: All Reolink IP cameras( Reolink E1 pro and E1 zoom included) except for battery-powered cameras and B800/D800/B400/D400/E1/ Reolink Lumus. 
https://support.reolink.com/hc/en-us/articles/360007010473-How-to-Live-View-Reolink-Cameras-via-VLC-Media-Player

**Reply by blumarten** [reply]
Thank you Martin. I did try the WebRTC integration but when trying to configure it I just get this pop up and I don't know what it means so i can't move past it

<img width="351" alt="Screenshot 2022-08-22 at 07 19 30" src="https://user-images.githubusercontent.com/52979960/185852380-0eaf9cd5-a2e2-4bf2-84a1-ff6c23f9de28.png">
.
